.drone.yml

```diff
@@ -1,129 +1,119 @@
 pipeline:
   test_build:
-    image: docker:1.11
+    image: docker:17.09.0-ce
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - apk add --no-cache bash sed grep bc coreutils
+      - apk add --no-cache bash sed grep bc coreutils curl
       - ./ci-build.sh
     when:
-      event: [push, pull_request, tag]
+      event: [push, pull_request]
   
   build_clamav:
```

Comment:
There is exact same step below triggered on tag. Could combine the two by modifying `event` to be both.

Reply:
`./ci-build` doesn't just build the containers, it build them, tests them and then stops and removes them (images included). I could change the script so that it does not remove the containers and have this pipeline trigger on everything (pushes, pull_requests and tags) though. Thoughts?

Reply:
Comment was for `build_clamav` step.

Reply:
Oops. I replied to your other comment though.